packages/xo-server/src/xo-mixins/remotes.js

```diff
@@ -123,6 +124,7 @@ export default class {
     return this._remotesInfo
   }
 
+  @debounceWithKey.decorate(10e3)
```

Comment:
The issue with these debounces is that it will add a big UI latency when adding/removing objects from the UI.

@julien-f What do you think? Do you see an easy solution for this?
(quick reminder: since we're merging legacy backups with backup-ng, we wanted to debounce the methods called by subscriptions which are triggered by legacy backup components to avoid perf issues for users that are using backup-ng only)

Reply:
Yes, it's difficult to add a properly managed cache (which should be invalidated when the inputs changes), otherwise this will introduce annoying latencies.

I think we should only debounce methods that have a perf impacts (ie reading the disk, compiling results, etc.)